Репост в группу

lalka2016 opened the pull request:
Решает ишуй #219. Добавляет репост в группу. Репост можно сделать только в той группе, где вы — администратор, репост в группу с просто открытой стеной сделать нельзя.
![изображение](https://user-images.githubusercontent.com/99399973/236799134-6965a909-af8e-46d5-b254-fbe05eb37228.png)
![изображение](https://user-images.githubusercontent.com/99399973/236799218-d440232b-366b-459e-a465-d77e6bc48014.png)

Так же репост в группу можно совершить при помощи API (параметр group_id)

Commit:
Repost to groups

maksalees reviewed (changes requested):
Все хорошо, но есть нюанс...

Commit:
Репост в группу где ты не создатель

Добавил репост в группу, где ты не создатель, а так же исправил стиль кода

Commit:
Исправил стиль кода в renderShare

Commit:
ownedClubs → writeableClubs

maksalees reviewed (approved):
Сойдет

veselcraft reviewed (approved):
https://github.com/openvk/openvk/assets/22993408/5f1422f0-500f-4a16-94e1-503f13084261

celestora reviewed (commented):
Под пивко пойдёт. На будущее: пользуйтесь ServiceAPI для реализации таких вот околоапишных приколов.

celestora reviewed (changes requested):
![image](https://github.com/openvk/openvk/assets/34442450/bf6684d7-02af-43dd-bb7b-b57de447e15c)

Commit:
Moved getWriteableClubs to serviceAPI

И добавил проверку наличия хоть одной доступной группы для репоста, если таковой нет, второй переключатель в окне репоста блокируется

celestora reviewed (changes requested):
Есть замечания по безопасности и производительности

Commit:
Fixes

Исправил уязвимость когда можно было сделать репост в группу с закрытой стеной, теперь при репосте меняется число на самом сайте и ещё что-то

celestora reviewed (changes requested):
![image](https://github.com/openvk/openvk/assets/34442450/22a9f925-c45f-44ca-8f74-83977fd31740)

Commit:
Коммит целесторы

celestora reviewed (approved):
lgtm

veselcraft reviewed (changes requested):
по форматированию траблы

Commit:
Пофиксил форматирование (возможно)

veselcraft reviewed (approved):
https://github.com/openvk/openvk/assets/22993408/3c78fa4d-2abb-44aa-a9d9-07af0d19391e
